editor/editor_system.cpp

```diff
@@ -894,6 +917,10 @@ Variant EditorSystem::saveBeforeQuit(const Event* event)
 Variant EditorSystem::createNewScene(const Event* event)
 {
 	const String& sceneName = Extract<String>(event->getData());
+	if (checkReservedName(sceneName))
+	{
+		return false;
```

Comment:
Add a warning here that tells the user about the collision with reserved name

Reply:
done